src/journal/journalctl.c

```diff
@@ -683,8 +683,9 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_VERIFY_KEY:
                         arg_action = ACTION_VERIFY;
-                        arg_verify_key = optarg;
+                        arg_verify_key = strdup(optarg);
                         arg_merge = false;
+                        memset(optarg, 0, strlen(arg_verify_key));
```

Comment:
Grr, I just noticed that the commit message (“overwrite it with null bytes”) no longer applies with this change… too late to change it now